### DISCUSSION TITLE
Failed to call API on server

### DISCUSSION THREAD

**Discussion by EmanueleGuidotti** [payloadcms/payload#8291] [Q&A]
### Link to reproduction

arkage.staging.arkage.it

### Describe the Bug

Hi there,

I’ve just started with Payload. I’ve hosted a simple website copying the website template on a Plesk server and it is up and running here, [https://arkage.staging.arkage.it/](https://arkage.staging.arkage.it/).
However, when I want to create the admin user [https://arkage.staging.arkage.it/admin/create-first-user](https://arkage.staging.arkage.it/admin/create-first-user) the system is looking for an API from localhost and I get an error: 
http://localhost:3000/api/users/me. While the first API works correctly https://arkage.staging.arkage.it/api/users/init.
Why I have one api working with the correct endpoint and the second one looking from localhost?

### To Reproduce

Copy the website template: `npx create-payload-app@latest my-project -t website`
Run `yarn install`
Run `yarn build`
Upload the `dist`, `build`, `media`, `package.json` and `.env` on remote Plesk
Connect to remote Plesk and launch `yarn install`
On remote Plesk launch the node server
Visit the `/admin` url

### Payload Version

"payload": "^2.0.0"

### Adapters and Plugins

db-mongodb

**Comment by jmikrut** [comment]
Hey @EmanueleGuidotti this is likely due to your `serverURL` being defined and pointed at `localhost:3000` whenever you run `payload build`. You should not have to set `serverURL` at all, so try and remove that from your config and build / deploy again.

This is not a problem with Payload itself though so I will convert this to a Q&A discussion and we can continue to help!

**Reply by EmanueleGuidotti** [reply]
Hi @jmikrut do you mean to change the `payload.config.ts`?
